src/main/scala/com/lunatech/lunabot/LunabotServlet.scala

```diff
@@ -20,10 +23,15 @@ import scala.util.{Failure, Success}
  */
 class LunabotServlet extends ScalatraServlet with JacksonJsonSupport {
 
-  val logger = LoggerFactory.getLogger(getClass)
+  // Sets up automatic case class to JSON output serialization, required by
+  // the JValueResult trait.
+  protected implicit val jsonFormats: Formats = DefaultFormats
 
-  protected implicit val jsonFormats: Formats = DefaultFormats.withBigDecimal
+  val logger = LoggerFactory.getLogger(getClass)
+  //load configuration from the resources location (default is application.conf)
+  val configFactory = ConfigFactory.load("application.conf")
```

Comment:
default location is this application.conf file. This means that even if we leave the load function without arguments it would be fine. So, is it necessary to have an environment variable or maybe  let the library check for default location ?